### PR TITLE
Fix label classifier.

### PR DESCRIPTION
**Pull request by rmmh**
This was broken by #5721.

### CODE REVIEW

File: gubernator/github/classifier.py [hunk 1]
```diff
@@ -115,16 +115,14 @@ def get_labels(events, labels=None):
...
                     label = body['label']
-                    if label not in labels:
-                        labels.append(label)
+                    if label['name'] not in labels:
+                       labels[label['name']] = label['color']
                 elif action == 'unlabeled':
-                    label = body['label']
-                    if label in labels:
-                        labels.remove(label)
+                    labels.pop(body['label']['name'], None)
```
Comment (BenTheElder) [inline gubernator/github/classifier.py]: should this be `del labels[body['name']]` ?

Comment (cjwagner) [reply gubernator/github/classifier.py]: This should work. `pop` is defined for dictionaries in python. Specifying `None` as a default is unnecessary though.

Comment (rmmh) [reply gubernator/github/classifier.py]: Using `del` or omitting `None` causes a KeyError-- the point is to remove a key whether or not it exists.

**Comment by rmmh** [comment]
Added test case!

**Comment by rmmh** [comment]
@BenTheElder ping

**Pull request merged**